### ISSUE TITLE
sid: BK early allocation SID

### ISSUE THREAD

**Issue by mcr** [core-wg/yang-cbor#111]
````
Section 7.6.3

   Early Allocations are made with a one-year period, after which they
   are expired.  [BCP100] indicates that at most one renewal may be
   made.  For the SID allocation a far more lenient stance is desired.

   1.  An extension of a referencing documents Early Allocation should
       update any referenced Early Allocations to expire no sooner than
       the referencing document.

   2.  The [BCP100] mechanism allows the IESG to provide a second
       renewal, and such an event may prompt some thought about how the
       collection of documents are being processed.

The first point is rather curious, since it can have no normative force
(this is not a BCP that would override BCP 100) and is merely a
continnation of how a more lenient stance is desired.  It's rather odd
to see it written in this way.  The second point seems to be the only
really useful part, and in practice this IESG approval of second (and
more!) renewals has become rather common, to the extent where a revision
of BCP 100 has been discussed to change the discussion of extensions for
early renewals.

   This is driven by the very generous size of the SID space and the
   often complex and deep dependencies of YANG modules.  Often a core
   module with many dependencies will undergo extensive review, delaying
   the publication of other documents.

Having many *dependencies* and taking a while shouldn't delay
publication of anything else; however, having many other modules
dependent on a core module would be likely to do so.

````

**Comment by ivajloip** [comment]
For me the whole point of a more lenient stance and still referencing BCP 100 might have been an error. While I understand that it would be difficult for some YANG modules to be ready in less than 2 years, they might not want to do the early allocation right after WG adoption. For me the options are:
1. Define a new process. It will take even more time and it might cause problems as it's going out of the way everybody understands.
2. Live with normal BCP 100 and accept that some documents will not have real values early on or their values will expire, but the experts might be able to keep them for some extra time, when that is judged appropriate.
3. Open the space 1,000 to 60,000 for everyone that wants it, as long as the expert reviewers accept that. For me this is not a great option as it might be difficult for experts to judge and push back against some registrations that will never be really used. In effect it will be allowing anyone to write a draft and advance it a bit and then you can get a range in the IETF space and forget about your draft. I would expect such a strategy to cause raise some questions.

Therefore, my personal preference is for option 2 - scratch this whole part.

**Comment by mcr** [comment]
solved by #120
